qnl/key-routing-service/src/main/java/com/uwaterloo/iqc/qnl/qll/cqptoolkit/server/KeyTransferServer.java

```diff
@@ -118,5 +117,17 @@ public void onKeyFromCQP(Key keyMessage, StreamObserver<Empty> responseObserver)
                 keyListener.onKeyGenerated();
             }
         }
+
+        @Override
+        public void onKeyFromCQP(Key keyMessage, StreamObserver<Empty> responseObserver) {
+            LOGGER.info("Received key from CQP.");
+            onKeyFromDriver(keyMessage, responseObserver);
+        }
+
+        @Override
+        public void onKeyFromSatellite(Key keyMessage, StreamObserver<Empty> responseObserver) {
+            LOGGER.info("Received key from satellite.");
```

Comment:
Remove the log. For each key, a line of log is written to the log file. The log will flood the log file quickly.

Reply:
Changed line to `debug`